src/FSharp.Compiler.Interactive.Settings/fsihelp.fs

```diff
@@ -0,0 +1,267 @@
+module FSharp.Compiler.Interactive.FsiHelp
+
+[<assembly: System.Runtime.InteropServices.ComVisible(false)>]
+[<assembly: System.CLSCompliant(true)>]
+do ()
```

Comment:
These assembly-level attributes are also in `fsiaux.fs`:

https://github.com/dotnet/fsharp/blob/a7a2d4ae2f88b62b7d2de778c605c299de158f30/src/FSharp.Compiler.Interactive.Settings/fsiaux.fs#L12-L14

But I think they only need to be in one place, so you could remove them from there, since this is now the first file in the project.

You could also just put them in an `AssemblyInfo.fs` file, or [add them to the fsproj](https://learn.microsoft.com/dotnet/standard/assembly/set-attributes-project-file#set-arbitrary-attributes):

```fsproj
<ItemGroup>
  <AssemblyAttribute Include="System.Runtime.InteropServices.ComVisible">
    <_Parameter1>false</_Parameter1>
  </AssemblyAttribute>
  <AssemblyAttribute Include="System.CLSCompliant">
    <_Parameter1>true</_Parameter1>
  </AssemblyAttribute>
</ItemGroup>
```

Reply:
I kept the single attributes in code as I'd like to leave it to the team to decide if we should put them in an `AssemblyInfo.fs or the fsproj file.